src/libs/ReportUtils.js

```diff
@@ -2051,6 +2061,11 @@ function getMoneyRequestOptions(report, reportParticipants, betas) {
         return [];
     }
 
+    // Additional requests should be blocked for settled reports
+    if (isIOUReport(report) && !report.hasOutstandingIOU) {
```

Comment:
Should we use the report status/ state for this? you can have a report with total  = 0 which is not settled but you can request money still
```suggestion
    if (isIOUReport(report) && !report.hasOutstandingIOU) {
```